wazo_sysconfd/http_server.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import multiprocessing
+
+from fastapi import FastAPI
+from gunicorn.app.base import BaseApplication
+
+api = FastAPI(title='wazo-sysconfd', openapi_url='/api/api.yml')
+
+
+def ensure_list(obj):
+    if not isinstance(obj, list):
+        return [obj]
+    else:
+        return obj
+
+
+class SysconfdApplication(BaseApplication):
+    def __init__(self, *args, config: dict = None, **kwargs):
+        self.config = config or {}
+        super().__init__(*args, **kwargs)
+
+    def load_config(self):
+        self.cfg.set(
+            'bind',
+            [
+                '{}:{}'.format(
+                    self.config['rest_api']['listen'], self.config['rest_api']['port']
+                )
+            ],
+        )
+        self.cfg.set('default_proc_name', 'sysconfd-api')
+        self.cfg.set('loglevel', self.config['log_level'])
+        self.cfg.set('accesslog', '-')
+        self.cfg.set('errorlog', '-')
+
+        try:
+            worker = multiprocessing.cpu_count() or 1
```

Comment:
I disagree with this:
* `cpu_count` does not return the [number of usable CPU](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.cpu_count)
* the number of workers should be fixed and configurable. A better default would be 8 or 10 workers, I guess.

Also, I suppose the number of workers is NOT the same as the max number of concurrent requests?

Reply:
Indeed, it is not. Each worker can handle a lot of requests, but if we want to use all cores we have to use gunicorn. Otherwise it is limited to a single core.